test/cmdlineTests/object_compiler/output

```diff
@@ -46,7 +46,6 @@ Text representation:
   0x00
     /* "object_compiler/input.yul":265:295   */
   return
-  pop
```

Comment:
Why was this emitted in the first place?

Reply:
To remove the local variable.

Reply:
After return? It is an unreachable pop.

Reply:
The code generator is unaware of control-flow.